docs/sources/alerting/monitor/_index.md

```diff
@@ -1,27 +1,166 @@
+
+#### grafana_alerting_alerts
+
+This metric is a counter that shows you the number of `normal`, `pending`, `alerting`, `nodata` and `error` alerts. For example, you might want to create an alert that fires when `grafana_alerting_alerts{state="error"}` is greater than 0.
+
+#### grafana_alerting_schedule_alert_rules
+
+This metric is a gauge that shows you the number of alert rules scheduled. An alert rule is scheduled unless it is paused, and the value of this metric should match the total number of non-paused alert rules in Grafana.
+
+#### grafana_alerting_schedule_periodic_duration_seconds_bucket
+
+This metric is a histogram that shows you the time it takes to process an individual tick in the scheduler that evaluates alert rules. If the scheduler takes longer than 10 seconds to process a tick then pending evaluations will start to accumulate such that alert rules might later than expected.
+
+#### grafana_alerting_schedule_query_alert_rules_duration_seconds_bucket
+
+This metric is a histogram that shows you how long it takes the scheduler to fetch the latest rules from the database. If this metric is elevated then so will `schedule_periodic_duration_seconds`.
+
+#### grafana_alerting_scheduler_behind_seconds
+
+This metric is a gauge that shows you the number of seconds that the scheduler is behind where it should be. This number will increase if `schedule_periodic_duration_seconds` is longer than 10 seconds, and decrease when it is less than 10 seconds. The smallest possible value of this metric is 0.
+
+#### grafana_alerting_notification_latency_seconds_bucket
+
+This metric is a histogram that shows you the number of seconds taken to send notifications for firing and resolved alerts. This metric will let you observe slow or over-utilized integrations, such as an SMTP server that is being given emails faster than it can send them.
+
+> These metrics are not available at present in Grafana Cloud.
+
+## Metrics for Mimir-managed alerts
+
+To meta monitor Grafana Mimir-managed alerts, open source and on-premise users need a Prometheus/Mimir server, or another metrics database to collect and store metrics exported by the Mimir ruler.
+
+#### rule_evaluation_failures_total
+
+This metric is a counter that shows you the total number of rule evaluation failures.
+
+## Metrics for Alertmanager
+
+To meta monitor the Alertmanager, you need a Prometheus/Mimir server, or another metrics database to collect and store metrics exported by Alertmanager.
+
+For example, if you are using Prometheus you should add a `scrape_config` to Prometheus to scrape metrics from your Alertmanager.
+
+### Example
+
+```yaml
+- job_name: alertmanager
+  honor_timestamps: true
+  scrape_interval: 15s
+  scrape_timeout: 10s
+  metrics_path: /metrics
+  scheme: http
+  follow_redirects: true
+  static_configs:
+    - targets:
+        - alertmanager:9093
+```
+
+### List of available metrics
+
```

Comment:
should have an intro here, like "The following sections..." otherwise you have stacked headings.

Reply:
thanks..updated